.eslintrc.yml

```diff
@@ -6,10 +6,14 @@ plugins:
 overrides:
   - files: '**/*.md'
     processor: 'markdown/markdown'
+env:
+  es2022: true
+  node: true
 rules:
   eol-last: error
   eqeqeq: ["error", "always", { "null": "ignore" }]
   indent: ["error", 2, { "MemberExpression": "off", "SwitchCase": 1 }]
   no-mixed-spaces-and-tabs: error
   no-trailing-spaces: error
+  no-unused-vars: [error, { vars: all, args: none, ignoreRestSiblings: true }]
   one-var: ["error", { "initialized": "never" }]
```

Comment:
This rule exists in this repo, but not in expressjs/express. I think it's ok. It disallows the following:

```js
var foo = 1,
     bar = 2;
```

and forces the following:

```js
var foo = 1;
var bar = 2;
```

Reply:
+1 to this :+1: